### PR TITLE
XW-201 | add spotify tests

### PR DESCRIPTION
**Pull request by tomasznapieralski**
https://wikia-inc.atlassian.net/browse/XW-201

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/SpotifyTests.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
+package com.wikia.webdriver.testcases.mercurytests.widgettests;
+
+import com.wikia.webdriver.common.contentpatterns.MercuryMessages;
+import com.wikia.webdriver.common.contentpatterns.MercurySubpages;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.mercury.NavigationSideComponentObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.widget.SpotifyWidgetPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.widget.VKWidgetPageObject;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/SpotifyTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.widget.VKWidgetPageObject'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by bognix** [comment]
remove unused import and LGTM

**Comment by bognix** [comment]
LGTM

**Pull request merged**